doc/build_instructions/arch_linux.md

```diff
@@ -4,7 +4,7 @@
 
 This command should provide required packages for Arch Linux installation:
 
-`sudo pacman -S --needed python python-pillow python-numpy python-pygments cython libepoxy ttf-dejavu freetype2 fontconfig harfbuzz cmake sdl2 sdl2_image opusfile opus-tools python-pylint qt5-declarative qt5-quickcontrols`
+`sudo pacman -S --needed python python-pillow python-numpy python-pygments cython libepoxy libpng15 ttf-dejavu freetype2 fontconfig harfbuzz cmake sdl2 sdl2_image opusfile opus-tools python-pylint qt5-declarative qt5-quickcontrols`
```

Comment:
should just be [`libpng`](https://www.archlinux.org/packages/extra/x86_64/libpng/)